### PR TITLE
Restructure "Resources", replace "Research" with "Readings" in Resources, and add dummy data for Resources

### PR DESCRIPTION
**Pull request by KHVBui**
### Mappings for Resource's dummy data:
Tutorials: id, title, description, url, tags
Readings: id, title, publicationDate, description, url, tags
Organizations: id, title, description, url, tags

### CODE REVIEW

**Comment by KHVBui** [comment]
Updates Resource.js to reference Tutorials, Readings, and Organizations js files instead of containing JSX itself.

**Comment by minhngo3818** [comment]
Can you add more data to each tab? It would be nice when there are at least 10 items per tab in order to test upcoming features including pagination, filter, (sort and search).

**Comment by minhngo3818** [comment]
Resolve issue #14

**Comment by KHVBui** [comment]
Yeah I can add more data to each tab.

File: src/resources/Readings.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './resources.css';
+
+
```
Comment (minhngo3818) [inline src/resources/Readings.js:4]: Please remove a blank line

**Review by minhngo3818 — APPROVED** [review]
Your PR looks good. Just remove a blank line after imports and you're good to go.

**Comment by KHVBui** [comment]
Yeah I need to totally reset this with all the prettier and linter changes and make sure it's all cleanup before I'll merge

**Review by minhngo3818 — APPROVED** [review]

**Pull request merged**